tests/testthat/test-preparedata-scatterplot.R

```diff
@@ -149,3 +149,152 @@ test_that("Handle y-values in multiple columns + multiple statistics",
     expect_equal(res5$scatter.variable.indices, c(x = 1, y = 2, sizes = 0, colors = 3, groups = 3))
     expect_equal(dim(res5$data), c(9, 5))
 })
+
+tables.with.banners <- list(X = structure(c(3.25318246110325, 10.8910891089109, 10.3253182461103,
```

Comment:
Could move this to an RDA file or at perhaps indent the file and truncate the numeric values to make it easier to read?

Reply:
I haven't moved this to an rda because when working on the code, it is sometimes helpful to manually change some of the elements in the input objects and play with that. However, I've moved all the input objects to the top of the test file so that the tests (starting from line 327) in test-preparedata-scatterplot.R is easier to read.